llvm/include/llvm/SYCLLowerIR/SYCLPropagateAspectsUsage.h

```diff
@@ -17,10 +17,22 @@
 
 namespace llvm {
 
+struct SPAUOptions {
+  SmallVector<StringRef, 8> TargetFixedAspects;
+};
```

Comment:
Yep, you're right. But since there is no assumed new options in the future, we can simplify it to StringRef. Or to add TODO and simplify it later, when the opional kernel feature implementation is ready and there is no new options.

Reply:
I would vote for simplifying it right away, because I'm not aware of any other similar options for that pass

Reply:
Okay, done [7ae0f0f](https://github.com/intel/llvm/pull/7115/commits/7ae0f0f3fa85a1e4882a51ad78a833cf13c16213)